genesis-programs/src/lib.rs

```diff
@@ -81,7 +81,7 @@ pub fn get_programs(operating_mode: OperatingMode, epoch: Epoch) -> Option<Vec<(
                 // The epoch of std::u64::MAX - 1 is a placeholder and is expected to be reduced in
                 // a future hard fork.
                 Some(vec![
-                    solana_config_program!(),
+                    solana_config_program::solana_config_program!(),
```

Comment:
This looks a little funny: `solana_config_program::solana_config_program!()`

Reply:
Yup, that is one of two changes I have in the wings, didn't want to kick ci for a couple of liittle nits.